### PR TITLE
Add score task action to Habitica

### PR DESCRIPTION
**Pull request by tr4nt0r**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/129605
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced two new actions for the Habitica integration: 
		- `habitica.score_habit` to adjust habit streaks (positive/negative).
		- `habitica.score_reward` to purchase custom rewards using gold.
- **Documentation**
	- Updated documentation with detailed descriptions and usage examples for the new actions.

### CODE REVIEW

File: source/_integrations/habitica.markdown [hunk 1]
```diff
@@ -130,6 +130,25 @@ Use a skill or spell from your Habitica character on a specific task to affect i
 
 To use task aliases, make sure **Developer Mode** is enabled under [**Settings -> Site Data**](https://habitica.com/user/settings/siteData). Task aliases can only be edited via the **Habitica** web client.
 
+### Action `habitica.score_habit`
+
+Increase the positive or negative streak of a habit
```
Comment (c0ffeeca7) [inline source/_integrations/habitica.markdown]:
```suggestion
Increase the positive or negative streak of a habit.
```

File: source/_integrations/habitica.markdown [hunk 2]
```diff
@@ -130,6 +130,25 @@ Use a skill or spell from your Habitica character on a specific task to affect i
...
+
+| Data attribute | Optional |  Description                                                                                                      |
+| -------------- | -------- | ----------------------------------------------------------------------------------------------------------------- |
+| `config_entry` | no       |  Config entry of the character for tracking the habit.                                                            |
```
Comment (c0ffeeca7) [inline source/_integrations/habitica.markdown]:
```suggestion
| `config_entry` | no       |  Config entry of the character tracking the habit.                                                            |
```

File: source/_integrations/habitica.markdown [hunk 3]
```diff
@@ -130,6 +130,25 @@ Use a skill or spell from your Habitica character on a specific task to affect i
...
 
+### Action `habitica.score_habit`
+
+Increase the positive or negative streak of a habit.
+
+| Data attribute | Optional |  Description                                                                                                      |
+| -------------- | -------- | ----------------------------------------------------------------------------------------------------------------- |
+| `config_entry` | no       |  Config entry of the character tracking the habit.                                                            |
+| `task`         | no       |  The name, `task ID`, or **alias** of the habit to track.                                                         |
+| `direction`    | no       |  `up` for positive progress or `down` for negative progress you want to track for your habit                      |
```
Comment (c0ffeeca7) [inline source/_integrations/habitica.markdown]:
```suggestion
| `direction`    | no       |  `up` for positive progress or `down` for negative progress you want to track for your habit.                     |
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @tr4nt0r :+1:

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @tr4nt0r 👋 

It seems like a merge conflict appeared 🙈 Could you take a look?

Thanks 👍 

../Frenck

**Comment by tr4nt0r** [comment]
> Hi there @tr4nt0r 👋
> 
> It seems like a merge conflict appeared 🙈 Could you take a look?
> 
> Thanks 👍
> 
> ../Frenck

@frenck I'm on it.

**Review by frenck — APPROVED** [review]
Thanks, @tr4nt0r 👍 

../Frenck

**Pull request merged**